proguard-common.pro

```diff
@@ -0,0 +1,10 @@
+# Keep everything
+-keep class * { *; }
+-dontshrink
+-dontobfuscate
+-dontoptimize
+
+# Optionally, keep annotations if needed
+-keepattributes *Annotation*
```

Comment:
Temporarily disabling all rules in common as optimisations per modules will be required